### PR TITLE
global: use inspire-schemas for countries

### PR DESCRIPTION
**Pull request by drjova**

### CODE REVIEW

**Comment by drjova** [comment]
Depends on https://github.com/inspirehep/inspire-schemas/pull/397

File: ui/yarn.lock [hunk 1]
```diff
@@ -5037,6 +5059,14 @@ insert-css@^2.0.0:
   resolved "https://registry.yarnpkg.com/insert-css/-/insert-css-2.0.0.tgz#eb5d1097b7542f4c79ea3060d3aee07d053880f4"
   integrity sha1-610Ql7dUL0x56jBg067gfQU4gPQ=
 
+inspire-schemas@^61.2.6-2-g90a8aaf:
+  version "61.2.6-2-g90a8aaf"
+  resolved "https://registry.yarnpkg.com/inspire-schemas/-/inspire-schemas-61.2.6-2-g90a8aaf.tgz#bb12cf7b20a0aabdb42d443f6aaed32de0ebae7b"
+  integrity sha512-tt3ZeV84xK+F+G7uoA4IMWN6HWTx+12uxoWhpgMlds6GYQpFziEgXBfccr6wXX9iYCdEKvQ8jvNIxCLIr5oK7g==
+  dependencies:
```
Comment (harunurhan) [inline ui/yarn.lock:5066]: that seems wrong, `inspire-schemas` shouldn't have any dependencies, I think all those should have been a `devDependency`

**Comment by drjova** [comment]
@harunurhan updated

File: ui/src/submissions/common/schemas/constants.js [hunk 2]
```diff
@@ -211,249 +212,8 @@ export const degreeTypeOptions = Object.keys(DEGREE_TYPE_TO_DISPLAY).map(
...
-  { value: 'Western Sahara' },
-  { value: 'Yemen' },
-  { value: 'Zambia' },
-  { value: 'Zimbabwe' },
-];
+export const countryOptions = Object.values(countryCodeToName).map(value => ({
+  value,
+}));
+
 export const countryValues = countryOptions.map(getValue);
```
[the remark this answers is not in the document]
Comment (drjova) [reply ui/src/submissions/common/schemas/constants.js]: @harunurhan done

**Pull request merged**